java/ql/src/semmle/code/java/frameworks/FastJson.qll

```diff
@@ -0,0 +1,26 @@
+import java
+
+
+/**
+ * The class `com.alibaba.fastjson.JSON`.
+ */
+class FastJson extends RefType {
+    FastJson() { 
+        this.hasQualifiedName("com.alibaba.fastjson", "JSON") or
```

Comment:
I think you should probably either change this predicate or adjust the comment to include `JSONObject` as well.